specs/_features/eip7594/das-core.md

```diff
@@ -195,27 +195,34 @@ def recover_matrix(partial_matrix: Sequence[MatrixEntry],
 
 ```python
 def get_data_column_sidecars(signed_block: SignedBeaconBlock,
-                             blobs: Sequence[Blob]) -> Sequence[DataColumnSidecar]:
+                             cells_and_kzg_proofs: Sequence[Tuple[
+        Vector[Cell, CELLS_PER_EXT_BLOB],
+        Vector[KZGProof, CELLS_PER_EXT_BLOB]]]) -> Sequence[DataColumnSidecar]:
+    """
+    Given a signed block and the cells/proofs associated with each blob in the
+    block, assemble the sidecars which can be distributed to peers.
+    
+    Since there is no method which converts cells to a blob, this method takes
+    cells/proofs instead of blobs so it can re-create sidecars after recovery.
```

Comment:
Hmm I wonder if this comment is only relevant to people who participated in this PR.

I think most people who read the function will not wonder "why does this function take cells/proofs instead of blobs". After all `DataColumnSidecar` contains cells, not blobs.

Reply:
Yeah good call! Fixed.